.github/workflows/ci.yaml

```diff
@@ -226,6 +226,7 @@ jobs:
           cd deps/finch-core && make clean
       - name: Build project
         run: |
+          git status
```

Comment:
is this still required?

Reply:
No, but I doesn't really hurt anything, and I don't want to re-trigger the tests on this large PR lol. I can remove it in a followup for sure though